docs/contract-interface/serialization-interface.md

```diff
@@ -35,6 +35,26 @@ pub fn sum_borsh(#[serializer(borsh)] a: u32, #[serializer(borsh)] b: u32) -> u3
 
 Where the `result_serializer(borsh)` annotation will override the default result serialization protocol from JSON to borsh and the `serializer(borsh)` annotations will override the parameter serialization.
 
+#### Example
+
+A simple demonstration of getting a [Borsh-serialized](https://borsh.io), base64-encoded value from a unit test:
+
+```rust reference
+https://github.com/mikedotexe/rust-status-message/blob/03781079d1716584d114bec294de00f5336cd20d/src/lib.rs#L106-L114
+```
+
+The following snippet shows a simple function that takes this value from a frontend or CLI. Note: this method doesn't have a return value, so the `#[result_serializer(borsh)]` isn't needed.
+
+```rust reference
+https://github.com/mikedotexe/rust-status-message/blob/03781079d1716584d114bec294de00f5336cd20d/src/lib.rs#L41-L55
```

Comment:
Formatting for this one has one too many closing bracket at the end. Might just be my OCD, but would look nice to not have it there

Reply:
good catch, fixed!